app/helpers/states_helper.rb

```diff
@@ -0,0 +1,59 @@
+module StatesHelper
+
+  def us_states
+    [
+      ['Alabama', 'Alabama'],
+      ['Alaska', 'Alaska'],
+      ['Arizona', 'Arizona'],
+      ['Arkansas', 'Arkansas'],
+      ['California', 'California'],
+      ['Colorado', 'Colorado'],
+      ['Connecticut', 'Connecticut'],
+      ['Delaware', 'Delaware'],
+      ['District of Columbia', 'District of Columbia'],
+      ['Florida', 'Florida'],
+      ['Georgia', 'Georgia'],
+      ['Hawaii', 'Hawaii'],
+      ['Idaho', 'Idaho'],
+      ['Illinois', 'Illinois'],
+      ['Indiana', 'Indiana'],
+      ['Iowa', 'Iowa'],
+      ['Kansas', 'Kansas'],
+      ['Kentucky', 'Kentucky'],
+      ['Louisiana', 'Louisiana'],
+      ['Maine', 'Maine'],
+      ['Maryland', 'Maryland'],
+      ['Massachusetts', 'Massachusetts'],
+      ['Michigan', 'Michigan'],
+      ['Minnesota', 'Minnesota'],
+      ['Mississippi', 'Mississippi'],
+      ['Missouri', 'Missouri'],
+      ['Montana', 'Montana'],
+      ['Nebraska', 'Nebraska'],
+      ['Nevada', 'Nevada'],
+      ['New Hampshire', 'New Hampshire'],
+      ['New Jersey', 'New Jersey'],
+      ['New Mexico', 'New Mexico'],
+      ['New York', 'New York'],
+      ['North Carolina', 'North Carolina'],
+      ['North Dakota', 'North Dakota'],
+      ['Ohio', 'Ohio'],
+      ['Oklahoma', 'Oklahoma'],
+      ['Oregon', 'Oregon'],
+      ['Pennsylvania', 'Pennsylvania'],
+      ['Puerto Rico', 'Puerto Rico'],
+      ['Rhode Island', 'Rhode Island'],
+      ['South Carolina', 'South Carolina'],
+      ['South Dakota', 'South Dakota'],
+      ['Tennessee', 'Tennessee'],
+      ['Texas', 'Texas'],
+      ['Utah', 'Utah'],
+      ['Vermont', 'Vermont'],
+      ['Virginia', 'Virginia'],
+      ['Washington', 'Washington'],
+      ['West Virginia', 'West Virginia'],
+      ['Wisconsin', 'Wisconsin'],
+      ['Wyoming', 'Wyoming']
+    ]
+  end
+end
```

Comment:
Instead of using the `select` method in your views, like so:
```
<%= f.select(:state, options_for_select(us_states)) %>
```
you could change your us_states method to return an array of states, and use the collection_select method:
```
<%= f.collection_select :state, us_states, :to_s, :to_s %>
```